frontend/src/index.css

```
@@ -891,6 +891,30 @@
     }
   }
 
+  @keyframes sparkleAnim {
```

Stylelint (17.9.0)

[error] Expected keyframe name "sparkleAnim" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)